src/js/entities-service/entities/program-actions.js

```diff
@@ -40,14 +40,11 @@ const _Model = BaseModel.extend({
   },
   getAction({ patientId, flowId }) {
     const currentUser = Radio.request('bootstrap', 'currentUser');
-    const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
-    const states = currentWorkspace.getStates();
 
     return Radio.request('entities', 'actions:model', {
       name: this.get('name'),
       _flow: flowId,
       _patient: patientId,
-      _state: states.at(0).id,
```

Comment:
It looks like this method is converting a program action into an action, so it would need a `state` assigned.